crates/katana/src/args.rs

```diff
@@ -135,9 +135,13 @@ pub struct StarknetOptions {
     pub total_accounts: u8,
 
     #[arg(long)]
-    #[arg(help = "Disable charging fee for transactions.")]
+    #[arg(help = "Disable charging fee when executing transactions.")]
     pub disable_fee: bool,
 
+    #[arg(long)]
+    #[arg(help = "Disable validation when executing transactions.")]
+    pub disable_validate: bool,
```

Comment:
i took the liberty to change this from `no_validate` to follow the naming convention of `disable_fee`

Reply:
Great change, thanks for that.